src/streamlink/plugins/zdf_mediathek.py

```diff
@@ -85,11 +84,11 @@ def _extract_streams(self, response):
             for format_ in priority["formitaeten"]:
                 yield self._extract_from_format(format_)
 
-    def _parse_track(self, track, parser):
+    def _parse_track(self, track, parser, name):
         try:
             return parser(self.session, track["uri"])
         except IOError as err:
-            self.logger.error("Failed to extract {0} streams: {1}", name, err)
+            self.logger.debug("Failed to extract {0} streams: {1}", name, err)
```

Comment:
**No, it's not worth it.**
This plugin is not for people who are geo blocked.

The **point of this PR** was to fix the http.get header error for people **who can use this plugin**,
not make it looks nice for people who can't even use this plugin.